src/LanguageLine.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
We don't use `strict_types` in our projects, so remove this

Reply:
Okay, but can I ask why's that? Isn't it better to use them?